spec/github_helper_spec.rb

```diff
@@ -15,7 +15,7 @@
     end
 
     before do
-      allow(described_class).to receive(:github_client).and_return(client)
+      described_class.instance_variable_set(:@client, client)
```

Comment:
Self-Review: I'm not so sure if this is the right way to do this, now that the OktoClient is an instance variable. If you have any suggestions for another approach, I appreciate it 😄

Reply:
If you declare the client as an `attr_reader :client` on `GithubHelper` (which, basically, code-generate a `def client; return @client; end` method under the hood for you), and then use `client` instead of `@client` in your implementation, then you'd be able to stub `receive(:client).and_return(client)` for it just like it was previously done for the `github_client` method before your changes.

Reply:
Thanks for the suggestion! I'll do it. 😄